packages/terra-heading/src/terra-dev-site/doc/example/HeadingSizes.jsx

```diff
@@ -8,6 +8,12 @@ const cx = classNames.bind(styles);
 
 const HeadingSizes = () => (
   <div>
+    <p>
```

Comment:
Don't put documentation inside the examples, move this it the MDX documentation above the example.